lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, from: nil, till: nil, outcome: nil, key: nil, primary: false)
```

Comment:
Zijn er betere namen voor de keys `from` en `till` te bedenken? De betekenis is mij een raadsel, zelfs na het lezen van de ticketomschrijving. Is het een `exists_from` en `exists_till` en dat het domein daarna irrelevant wordt? En wat is het verschil tussen `sbg_key` en `key`?